src/machines/java/com/enderio/machines/common/blockentity/PoweredSpawnerBlockEntity.java

```diff
@@ -44,8 +49,10 @@ public class PoweredSpawnerBlockEntity extends PoweredTaskMachineEntity<SpawnTas
     protected float bCol;
```

Comment:
Seems I have forgotten to turn the colour into a config like the vacuum blocks, but that should be done.

Reply:
I'd leave that to a future PR, this one's scope is already really wise